presentations/strings/7.rs

```diff
@@ -0,0 +1,9 @@
+fn main () {
+    let json = r##"
+{
```

Comment:
Maybe this block should be indented to line it up with the rest of the code. It looks tidier, but the indentation will be part of the string.

Reply:
I thought about it, too, but decided against it to show that multiline strings in Rust behave like heredocs in many other languages. A presenter should mention it, and I thought that this unpleasant formatting will prompt them to do so. I've added some text to a slide to mention it.